oteltest/harness.go

```diff
@@ -29,11 +29,15 @@ import (
 
 // Harness is a testing harness used to test implementations of the
 // OpenTelemetry API.
+//
+// Deprecated: this will be removed in the next major release.
 type Harness struct {
```

Comment:
Just type Harness = internaltest.Harness should be enough or embedding

Reply:
Ah, good point :+1: